extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -109,6 +109,14 @@ export const databaseName = localize('databaseName', "Database name");
 export const databaseVariable = localize('databaseVariable', "Database variable");
 export const serverName = localize('serverName', "Server name");
 export const serverVariable = localize('serverVariable', "Server variable");
+export const suppressMissingDependenciesErrors = localize('suppressMissingDependenciesErrors', "Suppress errors caused by unresolved references in the referenced project");
+export const exampleUsage = localize('exampleUsage', "Example Usage");
+export const enterSystemDbName = localize('enterSystemDbName', "Enter a database name for this system database");
+export const databaseNameRequiredVariableOptional = localize('databaseNameRequiredVariableOptional', "A database name is required. The database variable is optional.");
+export const databaseNameServerNameVariableRequired = localize('databaseNameServerNameVariableRequired', "A database name, server name, and server variable are required. The database variable is optional");
+export const sameDatabaseExampleUsage = 'SELECT * FROM [Schema1].[Table1]';
+export function differentDbSameServerExampleUsage(db: string) { return `SELECT * FROM [${db}].[Schema1].[Table1]"`; }
+export function differentDbDifferentServerExampleUsage(server: string, db: string) { return `SELECT * FROM [${server}].[${db}].[Schema1].[Table1]`; }
```

Comment:
Can you please put these in a new section in the file called "SQL queries" or something to separate them out from the localized strings?  Thanks!